etl/steps/data/garden/harvard/2023-09-18/colonial_dates_dataset.py

```diff
@@ -88,10 +92,21 @@ def process_data(tb: Table) -> Table:
     tb_rest = tb_rest.drop(columns=["colstart_max", "colend_max", "colstart_mean", "colend_mean", "col"])
 
     # Create another table with the total number of colonies per colonizer and year
-    tb_count = tb_colonized.groupby(["colonizer", "year"]).agg({"country": "count"}).reset_index().copy_metadata(tb)
+    tb_count = tb_colonized.copy()
+    tb_count = geo.harmonize_countries(df=tb_count, countries_file=paths.country_mapping_path)
+    tb_count = tb_count.merge(tb_pop[["country", "year", "population"]], how="left", on=["country", "year"])
+
+    tb_count = (
+        tb_count.groupby(["colonizer", "year"])
+        .agg({"country": "count", "population": "sum"})
+        .reset_index()
+        .copy_metadata(tb)
```

Comment:
I haven't looked in depth, but I'd say that here it should be `.copy_metadata(tb_count)`. I've run the steps with that change and they work.

Reply:
Or even better, I don't think you need to copy any metadata, but simply:
```
tb_count = tb_count.groupby(["colonizer", "year"], observed=True).agg({"country": "count", "population": "sum"}).reset_index()
```